helm/cluster-vsphere/README.md

```diff
@@ -145,6 +145,8 @@ Properties within the `.global.providerSpecific` object
 
 | **Property** | **Description** | **More Details** |
 | :----------- | :-------------- | :--------------- |
+| `global.providerSpecific.additionalInfrastructureAnnotations` | **Additional infrastructure annotations** - Additional annotations to be added to the infrastructure cluster.|**Type:** `object`<br/>**Default:** `{}`|
```

Comment:
these properties here are based on https://github.com/giantswarm/cluster-azure/blob/a1761aafb77582c6f70859a53ea1cdf26898edbf/helm/cluster-azure/README.md?plain=1#L216-L217
I used all the same properties

Reply:
I think in this case it might be better to make it more explicit like `azureResourceTags` or something like that. Otherwise infrastructure would suggest (at least to me) vSphere (which also has tags). And who knows we might need it for AWS one day too 😄